.changeset/eleven-women-glow.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/tree": minor
```

Comment:
Tree is intended to be used (by non legacy users) from the fluid framework package, so all non legacy changes to tree should include in in the changeset.

```suggestion
"fluid-framework": minor
"@fluidframework/tree": minor
```